mongo/mongo.go

```diff
@@ -0,0 +1,80 @@
+package mongo
+
+import (
+	"context"
+
+	"github.com/ONSdigital/dp-healthcheck/healthcheck"
+	"github.com/ONSdigital/dp-legacy-cache-api/config"
+	"github.com/ONSdigital/dp-legacy-cache-api/models"
+	mongoHealth "github.com/ONSdigital/dp-mongodb/v3/health"
+	mongoDriver "github.com/ONSdigital/dp-mongodb/v3/mongodb"
+	"github.com/ONSdigital/log.go/v2/log"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+type Mongo struct {
+	mongoDriver.MongoDriverConfig
+
+	Connection   *mongoDriver.MongoConnection
+	healthClient *mongoHealth.CheckMongoClient
+}
+
+func NewMongoStore(_ context.Context, cfg config.MongoConfig) (m *Mongo, err error) {
```

Comment:
All public functions should be preceeded with valid godoc
```suggestion
// NewMongoStore [does xyz]
func NewMongoStore(_ context.Context, cfg config.MongoConfig) (m *Mongo, err error) {
```
This applies to all other public functions, types and variables